### PR TITLE
update for v1.1.0

### PR DESCRIPTION
**Pull request by jthorton**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jthorton** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -17,15 +19,15 @@ build:
 requirements:
   host:
     - pip
-    - python >=3.9
+    - python {{ python_min }}
```
Comment (atravitz) [inline recipe/meta.yaml:22]:
```suggestion
    - python >= {{ python_min }}
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -35,6 +37,7 @@ test:
   requires:
     - pip
     - pytest
+    - python {{ python_min }}
```
Comment (atravitz) [inline recipe/meta.yaml:43]:
```suggestion
    - python >= {{ python_min }}
```

**Review by atravitz — COMMENTED** [review]
fixing min version syntax

**Comment by atravitz** [comment]
@conda-forge-admin, please rerender

**Comment by atravitz** [comment]
@conda-forge-admin, please rerender

**Pull request closed without merging**